front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/adjustConfWithTrainToModify.ts

```diff
@@ -1,28 +1,12 @@
-import {
-  updateLabels,
-  updateRollingStockID,
-  updateSpeedLimitByTag,
-  updateAllowances,
-  toggleUsingElectricalProfiles,
-  updateName,
-  updateDepartureTime,
-  updateInitialSpeed,
-  updatePathfindingID,
-  updateItinerary,
-  updateOrigin,
-  updateDestination,
-  replaceVias,
-  updateSuggeredVias,
-  updatePowerRestrictionRanges,
-} from 'reducers/osrdconf';
 import { sec2time } from 'utils/timeManipulation';
 import { Dispatch } from 'redux';
-import { store } from 'Store';
 import { Allowance, Path, TrainSchedule } from 'common/api/osrdEditoastApi';
 import { ArrayElement } from 'utils/types';
 import { PointOnMap } from 'applications/operationalStudies/consts';
 import { compact } from 'lodash';
 import { ms2kmh } from 'utils/physics';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Comment:
Could you add `type` to all other type imports in the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Reply:
Done